packages/contracts/test/contracts/utils/MerkleTrie.spec.ts

```diff
@@ -21,12 +21,11 @@ describe('MerkleTrie', () => {
 
   let MerkleTrie: ContractFactory
   before(async () => {
-    MerkleTrie = await ethers.getContractFactory('MerkleTrie')
+    MerkleTrie = await ethers.getContractFactory('MockMerkleTrie')
```

Comment:
Why are we using a MockMerkleTrie here? Sorry I really should look deeper into the code but just asking this because was surprised to see `MerkleTrie.spec` testing against a mock

Reply:
Hmmm maybe the word "mock" is confusing. I pulled the term from openzeppelin's primary contracts repo. `MockMerkleTrie` is just a standard contract that pipes all of the requests to `MerkleTrie`.